app/src/test/java/org/simple/clinic/home/overdue/search/OverdueSearchEffectHandlerTest.kt

```diff
@@ -104,4 +123,35 @@ class OverdueSearchEffectHandlerTest {
     verify(uiActions).openContactPatientSheet(patientUuid)
     verifyNoMoreInteractions(uiActions)
   }
+
+  @Test
+  fun `when search overdue patient effect is received, then search overdue patient`() {
+    // given
+    val query = "Ani"
+    val overdueAppointment = listOf(TestData.overdueAppointment(
+        facilityUuid = currentFacility.uuid,
+        name = "Anish Acharya",
+        patientUuid = UUID.fromString("37259e96-e757-4608-aeae-f1a20b088f09")
+    ), TestData.overdueAppointment(
+        facilityUuid = currentFacility.uuid,
+        name = "Anirban Dar",
+        patientUuid = UUID.fromString("53659148-a157-4aa4-92fb-c0a7991ae872")
+    ))
+
+    val expectedPagingData = PagingData.from(overdueAppointment)
+
+    whenever(pagerFactory.createPager(
+        sourceFactory = any<PagingSourceFactory<Int, OverdueAppointment>>(),
+        pageSize = eq(pagingLoadSize),
+        enablePlaceholders = eq(false),
+        initialKey = eq(null)
+    )) doReturn Observable.just(expectedPagingData)
+
+    // when
+    effectHandlerTestCase.dispatch(SearchOverduePatients(query, LocalDate.now()))
```

Comment:
Instead of `LocalDate.now` can we change it to a static date as we do with other tests, please?

This can cause flakiness in our tests.